src/librustdoc/html/render/print_item.rs

```diff
@@ -663,13 +663,16 @@ fn item_trait(w: &mut Buffer, cx: &Context<'_>, it: &clean::Item, t: &clean::Tra
             }
         }
 
-        let (local, foreign) = implementors.iter().partition::<Vec<_>, _>(|i| {
+        let (mut local, mut foreign) = implementors.iter().partition::<Vec<_>, _>(|i| {
             i.inner_impl()
                 .for_
                 .def_id_full(cx.cache())
                 .map_or(true, |d| cx.cache.paths.contains_key(&d))
         });
 
+        local.sort_by(|a, b| compare_impl(&a, &b, cx));
```

Comment:
I see that a few lines later `synthetic` and `concrete` (which come from `local`) are sorted as well. Is it necessary to sort in both places?

Also, a small nit: I'm wondering if these `&`s can be removed (and on the next line) since later on they aren't used when calling `compare_impl`.

Reply:
Done, thanks!